contract/Cargo.toml

```diff
@@ -2,21 +2,17 @@
 name = "contract"
 version = "0.1.0"
 authors = ["Karan Dhareshwar <karan@casperlabs.io>"]
```

Comment:
You should add yourself as an author as well! :)

Reply:
:) maybe I spent too much time in enterprises, but I think a general authors list would be easier to maintain. I've made the update to this:
> authors = ["CasperLabs <https://discord.com/invite/Q38s3Vh>"]